sympy/printing/tests/test_pycode.py

```diff
@@ -92,6 +105,26 @@ def test_SciPyPrinter():
     smat = SparseMatrix(2, 5, {(0, 1): 3})
     assert p.doprint(smat) == 'scipy.sparse.coo_matrix([3], ([0], [1]), shape=(2, 5))'
     assert 'scipy.sparse' in p.module_imports
+    assert p.doprint(erf(x)) == 'scipy.special.erf(x)'
+    assert p.doprint(erfc(x)) == 'scipy.special.erfc(x)'
+    assert p.doprint(besselj(x, y)) == 'scipy.special.jv(x, y)'
+    assert p.doprint(bessely(x, y)) == 'scipy.special.yv(x, y)'
+    assert p.doprint(besseli(x, y)) == 'scipy.special.iv(x, y)'
+    assert p.doprint(besselk(x, y)) == 'scipy.special.kv(x, y)'
+    assert p.doprint(factorial(x)) == 'scipy.special.factorial(x, exact=true)'
+    assert p.doprint(gamma(x)) == 'scipy.special.gamma(x)'
+    assert p.doprint(loggamma(x)) == 'scipy.special.gammaln(x)'
+    assert p.doprint(digamma(x)) == 'scipy.special.psi(x)'
+    assert p.doprint(RisingFactorial(x, y)) == 'scipy.special.poch(x, y)'
+    assert p.doprint(jacobi(1, x, y, z)) == 'scipy.special.eval_jacobi(1, x, y, z)'
```

Comment:
```suggestion
    assert p.doprint(jacobi(1, x, y, z, evaluate=False)) == 'scipy.special.eval_jacobi(1, x, y, z)'
```
This function automatically evaluates, so the only way to disable is like this way